src/drive/web/modules/navigation/duck/actions.jsx

```diff
@@ -164,6 +164,27 @@ export const fetchMoreFiles = (folderId, skip, limit) => {
     }
   }
 }
+/**
+ *
+ * @param {Array} files
+ * @param {Array} parentFolders
+ */
+export const mergeFilesWithParents = (files, parentFolders) => {
+  const filesWithPath = files.map(file => {
+    const parentFolder = parentFolders.rows.find(row => row.id === file.dir_id)
+    /* We can have FSCK's issue mostly index_orphan_tree or filesystem_missing.
+      In that case, we can receive an empty doc without path. So let's be
+      more defensive to not crash the recent view */
+    //!TODO Here we are able to detect an FSCK error. We should add a flag to the file
+    //to inform the user
+    const path =
+      parentFolder && parentFolder.doc && parentFolder.doc.path
+        ? parentFolder.doc.path
+        : ''
```

Comment:
Is the file shown to the user when its path is empty ? What is the meaning of having an empty path ? Should we have a sentinel value like FSCK_PATH_PROBLEM instead of empty string ?

Reply:
See the TODO... I don't want to do a one shot fix for this kind of error and we need to brainstorm on how to handle those cases.

Reply:
> Is the file shown to the user when its path is empty ? What is the meaning of having an empty path ? 

Described in the PR...